uniflow/flow/transform/transform_summary_flow.py

```diff
@@ -0,0 +1,161 @@
+"""Paper Comparison Flow Module."""
+
+from typing import Any, Dict, Sequence
+
+from uniflow.constants import TRANSFORM
+from uniflow.flow.flow import Flow
+from uniflow.node import Node
+from uniflow.op.model.lm.model import LmModel
+from uniflow.op.model.model_op import ModelOp
+from uniflow.op.prompt import PromptTemplate
+from uniflow.op.prompt import Context
+
+from uniflow.op.basic.expand_op import ExpandOp
+from uniflow.op.basic.reduce_op import ReduceOp
+from uniflow.op.basic.group_op import GroupOp
+
+
+class GoogleModelFlow(Flow):
```

Comment:
There is another flow called `GoogleModelFlow`, Shall we rename this to another name?

Reply:
should this be `SummaryGoogleFlow`.